src/main/java/mate/academy/spring/service/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.dto.request.CinemaHallRequestDto;
+import mate.academy.spring.dto.response.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+    public CinemaHallResponseDto parse(CinemaHall cinemaHall) {
```

Comment:
it would be better to rename this method in the all mappers
```suggestion
    public CinemaHallResponseDto toDto(CinemaHall cinemaHall) {
```